contracts/erc721/UserMintableERC721.sol

```diff
@@ -567,12 +572,31 @@ contract UserMintableERC721 is Initializable, ERC721, ERC721Enumerable, ERC721Me
     );
 
     // Extract the public key from the signature
+    bytes32 payload_ = sha256(abi.encodePacked(signingRoot, transitionValidated));
```

Comment:
As we are adding a new field to the signature and we can only recover from [32]byte length payloads we need to rehash it here. Same as the node.